src/components/App.tsx

```diff
@@ -58,40 +58,47 @@ function App() {
 
   // Handles page print
   const handlePrint = () => {
-    const css = '@page { size: landscape; }',
+    const css = '@page { size: A3 landscape; margin: 0; }',
       head = document.head || document.getElementsByTagName('head')[0],
       style = document.createElement('style');
-
     style.media = 'print';
-
     if ('styleSheet' in style) {
       const styleSheet = style.sheet as CSSStyleSheet;
       styleSheet.insertRule(css, styleSheet.cssRules.length);
     } else {
       style.appendChild(document.createTextNode(css));
     }
-
     head.appendChild(style);
-
     window.print();
   };
 
   // Event handler to set a common text for all proof elements
-  const setCommonText = () => {
+  const setAliceText = () => {
     const all = document.getElementsByClassName('proof');
     for (const elem of all) {
       elem.textContent = proofingText;
     }
   };
 
+  // Event handler to set a common text for all proof elements (for opentype)
+  const setOpentypeText = () => {
```

Comment:
Awesome work! This will definitely be necessary now and in the future :)

Reply:
Maybe it could be changed to incorporate Alice's text through  a parameter reference?